{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9bc1b8d916c02da07d4dc3ad875af475\")>\n\n    @allure.epic(\"VWO login Test\")\n    @allure.feature(\"TC#1 - VWO app positive Test case\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n        loginPage.login_to_vwo(usr=\"py2x@thetestingacademy.com\", pwd=\"Wingify@1234\")\n        time.sleep(10)\n        dashboardPage=DashboardPage(driver)\n        assert \"Dashboard\" in driver.title\n>       assert \"Aman\" in dashboardPage.get_user_logged_in_text()\n\nTestCase_VWO_login.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\PageObjects\\DashboardPage.py:18: in get_user_logged_in_text\n    return self.get_user_logged_in_text()\n..\\PageObjects\\DashboardPage.py:18: in get_user_logged_in_text\n    return self.get_user_logged_in_text()\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "start": 1715656507332, "stop": 1715656517729, "uuid": "24580640-bccc-4daa-af97-4592e47fe253", "historyId": "2f8ad64b01de3c9a5e779d92c322f34b", "testCaseId": "2f8ad64b01de3c9a5e779d92c322f34b", "fullName": "tests.VWO_LoginTests.TestCase_VWO_login#test_vwo_login_positive", "labels": [{"name": "feature", "value": "TC#1 - VWO app positive Test case"}, {"name": "epic", "value": "VWO login Test"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.VWO_LoginTests"}, {"name": "suite", "value": "TestCase_VWO_login"}, {"name": "host", "value": "INDELNB1516"}, {"name": "thread", "value": "28488-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.VWO_LoginTests.TestCase_VWO_login"}]}